IBAnimatable/IBEnum.swift

```diff
@@ -5,17 +5,19 @@
 
 import Foundation
 
+
+
 /**
  A protocol provides extension method for converting `String` into `enum`.
  Because `@IBInspectable` property can not support `enum` directly. To provide both `enum` API in code and `@IBInspectable` supported type `String` in Interface Builder, we use `IBEnum` to bridge Swift `enum` and `String`
  */
-public protocol IBEnum: StringLiteralConvertible {
+public protocol IBEnum {
   /**
    Initializes a swift `enum` with provided optional string
    
    - Parameter string: The optional string to be converted into `enum`.
    */
-  init(string: String?)
+  init?(string: String?)
```

Comment:
@lastMove I have tried out `IBOptionalEnum`. Please have a look at the commit https://github.com/IBAnimatable/IBAnimatable/commit/0b341304064e2858726b4718670515783573cb36 I think your solution is better 👍